checkov/arm/checks/resource/AppServiceSlotMinTLS.py

```diff
@@ -0,0 +1,21 @@
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+
+
+class AppServiceSlotMinTLS(BaseResourceValueCheck):
```

Comment:
Please remove this check from your PR, we already have this check for those resources in `CKV_AZURE_15` & `CKV_AZURE_145`